inputfiles/overridingTypes.json

```diff
@@ -1,5 +1,40 @@
 [
     {
+        "kind": "interface",
+        "interface": "CustomEventInit",
+        "typeParameters": [
+            "T = any"
+        ]
+    },
+    {
+        "kind": "property",
+        "interface": "CustomEventInit",
+        "name": "detail",
+        "type": "T"
+    },
+    {
+        "kind": "interface",
+        "interface": "CustomEvent",
+        "typeParameters": [
+            "T = any"
+        ]
+    },
+    {
+        "kind": "property",
+        "interface": "CustomEvent",
+        "readonly": true,
+        "name": "detail",
+        "type": "T"
+    },
+    {
+        "kind": "method",
+        "interface": "CustomEvent",
+        "name": "initCustomEvent",
+        "signatures": [
+            "(typeArg: string, canBubbleArg: boolean, cancelableArg: boolean, detailArg: T): void"
```

Comment:
you need to include the name here. i.e. `initCustomEvent(typeArg: string, canBubbleArg: boolean, cancelableArg: boolean, detailArg: T)`